packages/editor/src/components/properties/BoxColliderNodeEditor.tsx

```diff
@@ -22,7 +22,12 @@ export const BoxColliderNodeEditor: EditorComponentType = (props) => {
   const colliderComponent = getComponent(props.node.entity, ColliderComponent)
   const world = useWorld()
   const boxShape = world.physics.getRigidbodyShapes(colliderComponent.body)[0]
-  const isTrigger = isTriggerShape(boxShape)
+  const [isTrigger, setIsTrigger] = useState(isTriggerShape(boxShape))
+
+  const onUpdateTrigger = (value) => {
+    updateTrigger(props.node.entity, value)
```

Comment:
we dont need to call the new function `updateTrigger` here, we can just call `updateBoxCollider(props.node.entity, { isTrigger: value })`